library/src/scala/quoted/Quotes.scala

```diff
@@ -4314,6 +4340,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
       /** A dummy class symbol that is used to indicate repeated parameters
       *  compiled by the Scala compiler.
+      *
+      *  Used to represent the type `scala.<repeated>[T]` which is
+      *  a kind of sequence type of `T`s.
```

Comment:
"a kind of sequence type" ? Might be simpler to put an `@see Repeated`, though I'm not sure if scaladoc will proparly create a link to the `Repeated` doc that way.

Reply:
Updated